requirements.txt

```diff
@@ -18,7 +18,7 @@ statsmodels<=0.13.0
 simpleitk>=2.0.2,<3.0.0
 argschema>=3.0.1,<4.0.0
 glymur==0.8.19
-xarray<0.16.0
+xarray<=0.19.0
```

Comment:
Why less than? Was 0.16 still usable or is this just saying that it now works up to 0.19?

Reply:
pandas 1.5.0 requires 0.19.0 or later. I started by pinning to 0.19.0, but that is not available in python 3.6, which we still build against in CI.

I think people who think rightly about security do not like to leave open-ended requirements in case, for instance, xarray 0.20.0 develops some bug. I don't know how serious we are about actually following that policy.